distributed/tests/test_nanny.py

```diff
@@ -156,16 +156,6 @@ def test_nanny_alt_worker_class(c, s, w1, w2):
     assert w1.Worker is Something
 
 
-@pytest.mark.slow
-@gen_cluster(client=False, nthreads=[])
-def test_nanny_death_timeout(s):
-    yield s.close()
-    w = yield Nanny(s.address, death_timeout=1)
```

Comment:
I think that this test is still valid, but you just want to add something like the following:

```python
w = Nanny(s.address, death_timeout=1
with pytest.raises(gen.TimeoutError):
    yield w
```

Reply:
Got it, thanks.

I've removed my new tests, since I think they're duplicative of these now.